VTEX - Audience API.json

```diff
@@ -0,0 +1,320 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "version": "1.0.0",
+        "title": "Audience API",
+        "description": "API for managing audiences and mapping price tables to these audiences for contextual seller pricing.\r\n\r\n## Index\r\n\r\n### Audience Manager\r\n- `POST` [Fetch audience](https:\/\/developers.vtex.com\/docs\/api-reference\/audience-api\/#post-\/api\/audience-manager\/pvt\/audience)\r\n\r\n### Price Table Mapper\r\n- `GET` [Get price table mapping](https:\/\/developers.vtex.com\/docs\/api-reference\/audience-api\/#get-\/api\/price-table-mapper\/pvt\/mapping\/-audienceId-)\r\n- `PUT` [Set price table mapping](https:\/\/developers.vtex.com\/docs\/api-reference\/audience-api\/#put-\/api\/price-table-mapper\/pvt\/mapping\/-audienceId-)\r\n- `DELETE`[Delete price table mapping](https:\/\/developers.vtex.com\/docs\/api-reference\/audience-api\/#delete-\/api\/price-table-mapper\/pvt\/mapping\/-audienceId-)\r\n\r\n\r\n## Common parameters in the documentation\r\n\r\n| Parameter name | Description |\r\n| - | - |\r\n| `{{accountName}}`| Name of the VTEX account. Used as part of the URL. |\r\n| `{{environment}}`| Name of the VTEX account. Used as part of the URL. |\r\n| `{{X-VTEX-API-AppKey}}` | Unique identifier of the [application key](https:\/\/developers.vtex.com\/docs\/guides\/api-authentication-using-application-keys). |\r\n| `{{X-VTEX-API-AppToken}}` | Secret token of the [application key](https:\/\/developers.vtex.com\/docs\/guides\/api-authentication-using-application-keys). |"
```

Comment:
I think the introduction is missing a reference to existing documentation on audiences and price tables. Endpoint descriptions could benefit from that as well.